src/cow-react/types.ts

```diff
@@ -5,3 +5,10 @@ export type Milliseconds = number // Example: 30000 === 30 sec
 export type Writeable<T> = { -readonly [P in keyof T]: T[P] }
 
 export type Nullish<T> = T | null | undefined
+
+declare global {
+  interface Window {
+    fbq: any
+    lintrk: any
```

Comment:
Are there property names pre-defined?

Reply:
Those are the methods that these script provide and that we should call for tracking.
But to access them we need to use window object so that why we extend window types here.